gsd/gsd.c

```diff
@@ -630,7 +628,7 @@ uint32_t gsd_make_version(unsigned int major, unsigned int minor)
 int gsd_create(const char *fname, const char *application, const char *schema, uint32_t schema_version)
     {
     int extra_flags = 0;
-    #ifdef _WIN32
```

Comment:
Revert this change and the similar ones below to fix the failing test. This was fixed in #51.